internal/services/formatters/go/gosec/formatter.go

```diff
@@ -23,25 +23,45 @@ import (
 	"github.com/ZupIT/horusec/internal/enums/images"
 
 	"github.com/ZupIT/horusec-devkit/pkg/entities/vulnerability"
+	"github.com/ZupIT/horusec-devkit/pkg/enums/confidence"
 	"github.com/ZupIT/horusec-devkit/pkg/enums/languages"
+	"github.com/ZupIT/horusec-devkit/pkg/enums/severities"
 	"github.com/ZupIT/horusec-devkit/pkg/enums/tools"
 	"github.com/ZupIT/horusec-devkit/pkg/utils/logger"
-	dockerEntities "github.com/ZupIT/horusec/internal/entities/docker"
+	"github.com/ZupIT/horusec/internal/entities/docker"
 	"github.com/ZupIT/horusec/internal/helpers/messages"
 	"github.com/ZupIT/horusec/internal/services/formatters"
-	"github.com/ZupIT/horusec/internal/services/formatters/go/gosec/entities"
 )
 
+// issue represents the Issues schema from gosec output
+type issue struct {
+	Severity   severities.Severity   `json:"severity"`
+	Confidence confidence.Confidence `json:"confidence"`
+	Details    string                `json:"details"`
+	File       string                `json:"file"`
+	Code       string                `json:"code"`
+	Line       string                `json:"line"`
+	Column     string                `json:"column"`
+}
+
+// output represents the output schema from gosec.
+type output struct {
+	Issues []issue `json:"Issues"`
+}
+
+// Formatter represents the Gosec formatter.
```

Comment:
i think there is no need for the entities package too, but i personally prefer a file for structs and other for implementation. What do you think to move the structs declaration to a file like `entities.go`?

Reply:
Make sense, I'll change this.